fitbenchmarking/fitbenchmarking
# close plots when finished with

#### Description of Work

Fixes #227 


#### Testing Instructions

1. Run example_scripts/example_runScripts.py
2. Observe output: there should no longer be any complaints of too many plots open
